packages/state-transition/src/block/processWithdrawals.ts

```diff
@@ -203,3 +200,13 @@ export function getExpectedWithdrawals(
 
   return {withdrawals, sampledValidators: n, processedPartialWithdrawalsCount};
 }
+
+function getPartiallyWithdrawnBalance(withdrawals: capella.Withdrawal[], validatorIndex: ValidatorIndex): number {
+  let total = 0;
```

Comment:
I think we can do `let total = BigInt(0)`, so we only need to do `Number()` once at the end instead of repeatedly in the loop

Reply:
seems slightly faster, was curious if bigint addition has some penalty but does not seem to be relevant compared to reducing the type conversions